tripper/literal.py

```diff
@@ -0,0 +1,193 @@
+"""Literal rdf values."""
```

Comment:
Are there any changes in this content after transferring it here from `triplestore.py`? It might be nice to just to a PR (or a dedicated commit) that has _only_ the split up and move within it, and then a subsequent PR (or dedicated commit) to implement any changes.

Reply:
Yes, some bug fixing and adding parse_literal(). The PR could arguably have been cleaner if it was split up. I just ended up the possibility to do the refactoring while I anyway was messing around with tripper

Reply:
It's quite difficult/extra time consuming for me (or anyone) to review such a mixed change, since it requires one to go through the entire file from top to bottom as if it was a completely newly written file. The separation would have eased this, focusing in on only the real changes.